README.md

```diff
@@ -109,6 +110,12 @@ For more information please refer to the [search api documentation](https://gith
 
 See the [documentation on the Q&A frontend](docs/q-and-a.md) and how it relates to a finder.
 
+## Changing email signup page defaults
```

Comment:
I think rather than having this section,  it would be better to add a line under the advanced search screenshot, that has a line about the business finder and a link to the business finder documentation.

e.g.
The **business finder** provides a links-based facetted search of GOV.UK Brexit related content for businesses:
* [gov.uk/find-eu-exit-guidance-business](https://www.gov.uk/find-eu-exit-guidance-business)

[Read more about the business finder](...)

Reply:
Good idea.